Sources/Run/Services/StripeClient.swift

```diff
@@ -61,5 +61,149 @@ public class StripeClient {
         
         return nil
     }
+    
+    @discardableResult
+    private func postToStripe(payload: [String: Any],
+                              endpoint: String,
+                              subscriber: Subscriber) -> Response? {
+        do {
+            let url = "https://api.stripe.com/v1/\(endpoint)"
+            let data = try Jay().dataFromJson(anyDictionary: payload)
+            let finalJSON = try JSON(bytes: data)
+            let node = try Node(node: finalJSON)
+            let urlEncodedForm = Body.data(try! node.formURLEncoded())
```

Comment:
p0: Please don't use force unwrap. If value will be nil, app will be crashed. This is vary bad behavior use guard or default value instead.